app/controllers/admin_controller.rb

```diff
@@ -35,6 +36,9 @@ def export_str(shf_apps)
     out_str = export_header_str
 
     shf_apps.each do |m_app|
+
+      app_company = m_app.companies.last
```

Comment:
This would be a separate story but perhaps at some point we should refactor to ensure that _all_ companies associated with an application are processed.

Reply:
No, the other way around. One application can belong to many companies. See Fredrik Karlsson in the system for example.

Reply:
I guess they could be separated with | or something. 
cmp1 | cmp2 , next thing, paid | unpaid, next thing, 2019-01-01 | never paid, etc :)

Reply:
or have multiple lines:  1 line for the application and company 1, 2nd line with the application and company 2

Just depends on how you want to work with the spreadsheet.

Reply:
I was thinking that accommodating multiple companies / application could be done in another story (especially since it's relatively rare).  In fact, it would be a great story for a newbie who knows ruby and a little bit of rails (fun refactoring ruby logic - maybe even do something with recursion (woohoo !!!).